ticTacToe.js

```diff
@@ -808,35 +544,112 @@ function newGameComputerFirst() {
             borderWidth: "1px"
     }, 150 );
     
-    s[0] = 0;
-    s[1] = 0;
-    s[2] = 0;
-    s[3] = 0;
-    s[4] = 0;
-    s[5] = 0;
-    s[6] = 0;
-    s[7] = 0;
-    s[8] = 2;
-    
-    $("td#0").html("&nbsp;");
-    $("td#1").html("&nbsp;");
-    $("td#2").html("&nbsp;");
-    $("td#3").html("&nbsp;");
-    $("td#4").html("&nbsp;");
-    $("td#5").html("&nbsp;");
-    $("td#6").html("&nbsp;");
-    $("td#7").html("&nbsp;");
-    $("td#8").html("O");
+    turnCount = 0
+    disableLite()
+    resetState()
+    resetBoardUI()
+    updateBoard(8, "O")
+    changeState(8, 2)
+}
+/* istanbul ignore next */
+function newLiteGamePlayerFirst() {
+    $("td").animate({
+        fontSize: "30px",
+        borderWidth: "1px"
+    }, 150 );
+    $("table").animate({
+            borderWidth: "1px"
+    }, 150 );
+
+    turnCount = 0
+    enableLite()
+    resetState()
+    resetBoardUI()
+}
+/* istanbul ignore next */
+function newLiteGameComputerFirst() {
+    $("td").animate({
+        fontSize: "30px",
+        borderWidth: "1px"
+    }, 150 );
+    $("table").animate({
+            borderWidth: "1px"
+    }, 150 );
+
+    enableLite()
+    resetState()
+    resetBoardUI()
+    turnCount = 0
+    performMove(8, "O", 2)
+}
+
+/* istanbul ignore next */
+function executeTurnCycle(index) {
+    const avaliable = 0;
+    if(getStateAtIndex(index) === avaliable){
+        if (numPlayers === 2){
+            performMove(index, getPlayerToken(), getPlayerIndicator())
+        }
+
+        if(numPlayers === 1){
+            performMove(index, "X", 1)
+            if(!isGameOver()) {
+                performMove(computerMove(), "O", 2)
+            }
+        }
+    }
+}
```

Comment:
A lot of nesting going on here. Can you use return statements to control flow here and reduce that perhaps?

Reply:
Addressed in dbab173